docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+= Resolving issues
+
+You can resolve issues by using a Quick Fix (automatic code replacement) or by editing the code.
+
+.Procedure
+
+. Resolve issues detected by the {ProductShortName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), right-click the issue and select *Preview Quick Fix*. To accept the suggested fix, right-click the issue and select *Apply Quick Fix.*
```

Comment:
Each bullet seems to have multiple steps. I recommend you consider breaking them out into separate procedures or as subprocedures.